sycl/doc/extensions/GroupAlgorithms/SYCL_INTEL_group_algorithms.asciidoc

```diff
@@ -181,6 +182,15 @@ It is undefined behavior for any of these functions to be invoked within a +para
 |Return +true+ if _pred_ returns +true+ for no element in the range [_first_, _last_). _first_, _last_ and _pred_ must be the same for all work-items in the group.
 |===
 
+==== Elect
+
+|===
+|Function|Description
+
+|+template <typename Group> bool leader(Group g);+
+|Return +true+ if the calling work-item is the elected leader of the group.
```

Comment:
I think this description is quite vague...
What does "elected' mean?
"leader of the group" is not defined in the spec, so we probably should clarify:
- group -> work-group
- specify some properties of the leader (e.g. there is only one leader per work-group).

I'd like someone who works with specs to review.
I'm okay to merge if @mkinsner approves.

Reply:
Thanks, @bader.  That's a fair point.  I'll expand the description here to explain what a leader is.

I think we don't want to replace "group" with "work-group" here, though.  Our current implementation of these functions only supports `sycl::group` as an argument, but the description of the functionality should apply to any class representing a group of work-items.